plugin/src/test/groovy/org/gradle/testretry/FilterFuncTest.groovy

```diff
@@ -56,14 +56,14 @@ class FilterFuncTest extends AbstractGeneralPluginFuncTest {
         then:
         noRetry.each { testName ->
             with(result.output) {
-                it.count("acme.${testName} > flakyTest FAILED") == 1
```

Comment:
💬 Gradle 8.7 and older reports `acme.ExcludedTest`. Gradle 8.8+ reports `ExcludedTest`